endstate_correction/protocol.py

```diff
@@ -192,14 +193,15 @@ def perform_endstate_correction(protocol: Protocol) -> Results:
             dE_reference_to_target = np.array(dEs / kBT)  # remove units
             r.dE_reference_to_target = dE_reference_to_target
 
+        # from target to reference potential
         if protocol.target_samples is not None:  # if target samples are provided
             print("Performing FEP from target to reference potential")
             dEs, _, _ = perform_switching(
                 sim,
                 lambdas=np.flip(
                     list_of_lambda_values
-                ),  # NOTE: we reverse the list of provided lamba values to indicate switching from the reference to the target potential
-                samples=protocol.reference_samples,
```

Comment:
🥇 ah, thanks for finding this! That would have made a lot of problems and would have been hard to diagnose!